packages/plus/src/components/ContentCard/index.tsx

```diff
@@ -0,0 +1,199 @@
+
+const activeStyle = (theme: Theme) => css`
+  &:hover {
+    border: 1px solid ${theme.colors.primary.borderHover};
+    box-shadow: ${theme.shadows.defaultShadow};
+    cursor: pointer;
+  }
+`
+
+const Card = styled.div<{
+  onClick?: ContentCardProps['onClick']
+  href?: ContentCardProps['href']
+}>`
+  display: block;
+  text-align: left;
+  padding: 0;
+  text-decoration: none;
+  border: 1px solid ${({ theme }) => theme.colors.neutral.border};
+  border-radius: ${({ theme }) => theme.radii.default};
+  background: ${({ theme }) => theme.colors.neutral.background};
+  ${({ onClick, href, theme }) =>
+    onClick || href ? activeStyle(theme) : null};
+`
+
+const IconContainer = styled.div`
+  display: flex;
+  width: fit-content;
+  background: ${({ theme }) => theme.colors.neutral.backgroundWeak};
+  padding: ${({ theme }) => theme.space['1']};
+  border-radius: ${({ theme }) => theme.radii.default};
+`
+
+const StyledIconStack = styled(Stack, {
+  shouldForwardProp: prop => prop !== 'direction',
+})<{ direction: ContentCardProps['direction'] }>`
+  padding: ${({ theme, direction }) =>
+    direction === 'column'
+      ? `0 ${theme.space['3']} ${theme.space['3']} ${theme.space['3']}`
+      : `${theme.space['3']} ${theme.space['3']} ${theme.space['3']} 0`};
+`
+
+const SubContainer = styled(Stack, {
+  shouldForwardProp: prop => !['direction', 'href'].includes(prop),
+})<{
+  direction: ContentCardProps['direction']
+  href: ContentCardProps['href']
+}>`
+  padding: ${({ theme, direction }) =>
+    direction === 'column'
+      ? `${theme.space['3']} ${theme.space['3']} 0 ${theme.space['3']}`
+      : `${theme.space['3']} 0 ${theme.space['3']} ${theme.space['3']}`};
+  padding: ${({ theme, href }) => (!href ? `${theme.space['3']}` : null)};
+`
+
+const Image = styled('img', {
+  shouldForwardProp: prop => prop !== 'direction',
+})<{ direction: ContentCardProps['direction'] }>`
+  object-fit: cover;
+  border-radius: ${({ theme, direction }) =>
+    `${
+      direction === 'column'
+        ? `${theme.radii.default} ${theme.radii.default} 0 0`
+        : `${theme.radii.default} 0 0 ${theme.radii.default}`
+    }`};
+`
+
+type ContentCardProps = {
+  direction?: 'row' | 'column'
+  /**
+   * The image to display at the top of the card (if direction is column) or to the left of the card (if direction is row).
+   * By default, the image will be cropped to fit the card.
+   */
+  image?: string
+  /**
+   * The icon the second element of the card to be displayed after the image. We allow any ReactNode but recommend
+   * using `<ProductIcon>` from `@ultraviolet/icons`.
+   */
+  icon?: ReactNode
+  subtitle?: string
+  title: string
+  description?: string
+  children?: ReactNode
+  /**
+   * The href to link the card to. If not provided, the card will not be clickable.
+   */
+  href?: HTMLAnchorElement['href']
+  target?: HTMLAnchorElement['target']
+  onClick?: MouseEventHandler<HTMLElement>
+  loading?: boolean
```

Comment:
should we forward className in case of extra styling?

Reply:
@vincentaudebert Right! I forgot about it, thanks!